src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConstraintValidatorsPass.php

```diff
@@ -24,6 +24,7 @@ public function process(ContainerBuilder $container)
 
         $validators = array();
         foreach ($container->findTaggedServiceIds('validator.constraint_validator') as $id => $attributes) {
+            $validators[$id] = $id;
```

Comment:
I'll modify it but I still have a question for when we deprecate aliases for validators (if we ever do it):

What if there are multiple validator services configured using the same class? This is the same problem as in #16615, so the same choice may apply.